hazelcast/src/main/java/com/hazelcast/jet/impl/connector/StreamEventJournalP.java

```diff
@@ -344,39 +352,40 @@ private CompletableFuture<? extends ReadResultSet<? extends T>> readFromJournal(
                                  .toCompletableFuture();
     }
 
-    private static class ClusterMetaSupplier<E, T> implements ProcessorMetaSupplier {
+    public static HzClientDataStoreFactory getDataStoreFactory(HazelcastInstance hazelcastInstance, String name) {
```

Comment:
DataStore -> DataLink, same in message below

Reply:
Ok